src/GraphQL.StarWars/GraphQL.StarWars.csproj

```diff
@@ -1,14 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <Version>1.0.0</Version>
     <TargetFrameworks>netstandard1.1;net45</TargetFrameworks>
     <PackageTargetFallback>dnxcore50</PackageTargetFallback>
     <AssemblyName>GraphQL.StarWars</AssemblyName>
     <PackageId>GraphQL.StarWars</PackageId>
+    <PackageProjectUrl>https://github.com/graphql-dotnet/graphql-dotnet</PackageProjectUrl>
+    <PackageLicenseUrl>https://github.com/graphql-dotnet/graphql-dotnet/blob/master/LICENSE.md</PackageLicenseUrl>
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\GraphQL\GraphQL.csproj" />
+    <PackageReference Include="GraphQL" Version="2.0.0-alpha-820" />
```

Comment:
Hi there! Unfortunately it looks like this change broke the [GraphiQLCore project](https://github.com/graphql-dotnet/graphql-dotnet/blob/4d4f59dfe61a168352f714faaec4c10e12873e5f/src/GraphQL.GraphiQLCore/GraphQL.GraphiQLCore.csproj#L8), which still relies on a project reference.

Reply:
https://github.com/graphql-dotnet/example-aspnetcore This repo now replaces that project.  It will be removed from the source soon.

Reply:
Hah, just saw this comment... never mind the PR I just opened :)